src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,101 @@
+$card-width: 200px;
+$color-text-darkblue: #060b35;
+$color-main-blue: #00acdc;
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+
+  &_image {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__info-link:link {
+    width: 160px;
+    height: 134px;
+    text-decoration: none;
+  }
+
+  &__title {
+    color: $color-text-darkblue;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__id {
+    color: #616070;
+    font-size: 10px;
+    font-weight: 400;
+    margin: 0 0 17px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 24px;
+  }
+
+  &__rating-reviews {
+    font-size: 10px;
+    color: $color-text-darkblue;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-word {
+    color: #616070;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 16px;
+  }
+
+  &__price-value {
+    color: $color-text-darkblue;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 18px;
+    text-align: right;
+    margin: 0;
+  }
+
+  &__button {
+    display: flex;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: #fff;
+    height: 40px;
+    background: $color-main-blue;
+    border-radius: 5px;
+    text-decoration: none;
+  }
+
+  &__button:hover {
+    color: $color-main-blue;
+    background-color: #fff;
+    border: 1px solid $color-main-blue;
```

Comment:
Set border to the initial button state and change only color on hover
Use nesting here